Src/WitsmlExplorer.Api/Workers/CompareLogDataWorker.cs

```diff
@@ -88,6 +88,10 @@ public override async Task<(WorkerResult, RefreshAction)> Execute(CompareLogData
 
                 for (int i = 0; i < allMnemonics.Count; i++)
                 {
+                    if (cancellationToken != null && cancellationToken.Value.IsCancellationRequested)
```

Comment:
Can't all of this just be replaced by `cancellationToken?.ThrowIfCancellationRequested();`?